WordPress/Classes/Utility/BuildInformation/BuildConfiguration.swift

```diff
@@ -1,23 +1,18 @@
 enum BuildConfiguration: String {
-    /// Development build, usually run from Xcode
+    /// Development build, usually run from Xcode.
     case localDeveloper
 
-    /// Continuous integration builds for Automattic employees to test branches & PRs
-    case a8cBranchTest
+    /// Preproduction builds for Automattic employees.
+    case alpha
 
-    /// Beta released internally for Automattic employees
-    case a8cPrereleaseTesting
-
-    /// Production build released in the app store
+    /// Production build released in the app store.
     case appStore
 
     static var current: BuildConfiguration {
         #if DEBUG
             return .localDeveloper
         #elseif ALPHA_BUILD
-            return .a8cBranchTest
-        #elseif INTERNAL_BUILD
-            return .a8cPrereleaseTesting
```

Comment:
Based on the order of compiler flags, unless there is some more magic outside of the parts of the build system specific in the repo, this was never used.

Both `ALPHA_BUILD` and `INTERNAL_BUILD` used to be specified for the alpha build, which I assume was the result of the incomplete refactor.

```
OTHER_SWIFT_FLAGS = "$(inherited) -D ALPHA_BUILD -D INTERNAL_BUILD";
```